core/lib/state/src/rocksdb/mod.rs

```diff
@@ -131,23 +173,111 @@ impl RocksdbStorage {
         tracing::info!(
             "Secondary storage for L1 batch #{latest_l1_batch_number} initialized, size is {estimated_size}"
         );
+
+        self.save_missing_enum_indices(conn).await;
+    }
+
+    async fn save_missing_enum_indices(&self, conn: &mut StorageProcessor<'_>) {
+        if let (Some(start_from), true) = (
+            self.enum_migration_start_from(),
+            self.enum_index_migration_chunks > 0,
+        ) {
+            let started_at = Instant::now();
+            let chunk_from = u16::from_be_bytes(start_from);
+            let chunk_to = chunk_from.saturating_add(self.enum_index_migration_chunks - 1);
+            tracing::info!("RocksDB enum index migration is not finished, processing chunks {chunk_from}..={chunk_to}");
```

Comment:
will we immediately see the total number of chunks? It's always good to see the progress, let's report is somehow. Grafana might be an overkill.

Or maybe you do report it and I missed it

Reply:
Total number of chunks is 65536 by design. Anyway, I'm going to change approach according to https://github.com/matter-labs/zksync-era/pull/162#discussion_r1347307942, and will leave progress reported in logs